templates/index.html

```diff
@@ -71,6 +71,17 @@ <h1>List-Building Comparison</h1>
 								</label>
 								{% endif %}
 							</div>
+							<div class="col col1">
```

Comment:
Because adding another column here, will want to change Line 51 from `<div class="cols cols3">` to `<div class="cols cols4">` so the space is divided appropriately